core/src/main/targetconfigs/vz/vz-application.yaml

```diff
@@ -42,74 +40,73 @@ metadata:
   namespace: {{{namespace}}}
 spec:
   workload:
-    apiVersion: weblogic.oracle/v8
-    kind: Domain
-    metadata:
-      name: {{{domainPrefix}}}-domain
-      namespace: {{{namespace}}}
+    apiVersion: oam.verrazzano.io/v1alpha1
+    kind: VerrazzanoWebLogicWorkload
     spec:
-      domainUID: {{{domainUid}}}
+      template:
+        metadata:
+          name: {{{domainPrefix}}}-domain
+          namespace: {{{namespace}}}
+        spec:
+          domainUID: {{{domainUid}}}
 
-      # WebLogic Image Tool provides domainHome, domainHomeSourceType, and imageName
-      domainHome: {{{domainHome}}}
-      domainHomeSourceType: {{{domainHomeSourceType}}}
-      image: {{{imageName}}}
+          # WebLogic Image Tool provides domainHome, domainHomeSourceType, and imageName
+          domainHome: {{{domainHome}}}
+          domainHomeSourceType: {{{domainHomeSourceType}}}
+          image: {{{imageName}}}
```

Comment:
Should this and subsequent section aligned with previous line?

Reply:
This correctly aligned at the same level as domainUID

Reply:
This is what the generated output looks like:
```
apiVersion: core.oam.dev/v1alpha2
kind: Component
metadata:
  name: base-domain-domain
  namespace: base-domain
spec:
  workload:
    apiVersion: oam.verrazzano.io/v1alpha1
    kind: VerrazzanoWebLogicWorkload
    spec:
      template:
        metadata:
          name: base-domain-domain
          namespace: base-domain
        spec:
          domainUID: base-domain

          # WebLogic Image Tool provides domainHome, domainHomeSourceType, and imageName
          domainHome: /u01/domains/base_domain
          domainHomeSourceType: FromModel
          image: phx.ocir.io/stevengreenberginc/rdonat/todo:22

          imagePullSecrets:
            - name: base-domain-registry-credentials
          includeServerOutInPodLog: true
          webLogicCredentialsSecret:
            name: base-domain-weblogic-credentials
          configuration:
            introspectorJobActiveDeadlineSeconds: 900
            model:
              configMap: base-domain-configmap
              domainType: WLS

              # WebLogic Image Tool provides modelHome
              modelHome: /u01/wdt/models

              # encryption for the WDT model and the SystemSerializedIni.data file.
              # used only for model-in-image deployment, can be removed for other types.
              runtimeEncryptionSecret: base-domain-runtime-encryption-secret

            secrets:
              - base-domain-jdbc-tododb

          serverPod:
            env:
              - name: JAVA_OPTIONS
                value: "-Dweblogic.StdoutDebugEnabled=false"
              - name: USER_MEM_ARGS
                value: "-Djava.security.egd=file:/dev/./urandom -Xms64m -Xmx256m "
```

Reply:
never mind, viewing the actual file is ok